docs/specification/2025-03-26/basic/authorization.mdx

```diff
@@ -8,236 +8,344 @@ title: Authorization
 
 ### 1.1 Purpose and Scope
 
-The Model Context Protocol provides authorization capabilities at the transport level,
-enabling MCP clients to make requests to restricted MCP servers on behalf of resource
-owners. This specification defines the authorization flow for HTTP-based transports.
-
-### 1.2 Protocol Requirements
+The Model Context Protocol provides authorization capabilities at the transport level, enabling MCP
+clients to make requests to restricted MCP servers on behalf of resource owners. This specification
+defines the authorization flow for HTTP-based transports.
+
+The scope of this document is **user authorization** - how a client can authenticate a user to access a
+server and the server determine whether to authorize the user to access its tools, resources, or any
+other capabilities.
+
+This document **does not discuss server-to-server authorization patterns**.
+
+### 1.2 Baseline tenets
+
+1. **MCP server developers are not security experts**. The current specification assumes that developers
```

Comment:
it seems weird to mention the current spec this way in this PR? Shouldn't the PR remove this language because this PR is supposed to solve this problem?

Reply:
Good point, I will adjust.